code/clue.js

```diff
@@ -1,64 +1,275 @@
-  const theKiller = document.getElementById("killer")
-  const theKillerName = document.getElementById("killerName")
+    // This will change the background color of the card to the one connected to the chosen killer and show the full d of the killer.
+    const theKiller = document.getElementById("killer")
+    const theKillerName = document.getElementById("killerName")
+    const theKillerAge = document.getElementById("killerAge")
+    const theKillerOccupation = document.getElementById("killerOccupation")
+    const thekillerDescription = document.getElementById("killerDescription")
+    const theKillerImage = document.getElementById("killerImage").src = mystery.killer.image
+
+    theKiller.style.background = mystery.killer.color
+    theKillerName.innerHTML = "Name: " +
+        mystery.killer.firstName + " " + mystery.killer.lastName
+    theKillerAge.innerHTML = "Age: " + mystery.killer.age
+    theKillerOccupation.innerHTML = "Occupation: " + mystery.killer.occupation
+    thekillerDescription.innerHTML = "Description: " + mystery.killer.description
+    theKillerImage.innerHTML = mystery.killer.image
 
-  theKiller.style.background = mystery.killer.color
-  theKillerName.innerHTML =
-    mystery.killer.firstName + " " + mystery.killer.lastName
 }
 
 // CREATE FUNCTIONS pickWeapon and pickRoom in a similar way.
+const pickWeapon = () => {
+    weaponPicked = true;
+    // This will randomly select a weapon from the weapons. And add that to the mystery object.
+    mystery.weapon = randomSelector(weapons)
+
+    // This will show the name and the weight of the weapon.
+    const theWeapon = document.getElementById("weapon")
+    const theWeaponName = document.getElementById("weaponName")
+    const theWeaponWeight = document.getElementById("weaponWeight")
+    const theWeaponImage = document.getElementById("weaponImage").src = mystery.weapon.image
+
+
+    theWeapon.style.background = mystery.weapon.color
+    theWeaponName.innerHTML = "Weapon: " + mystery.weapon.name
+    theWeaponWeight.innerHTML = "Weight: " + mystery.weapon.weight + " kg"
+    theWeaponImage.innerHTML = mystery.weapon.image
+
+}
+
+const pickRoom = () => {
+    roomPicked = true;
+    // This will randomly select a room from the rooms. And add that to the mystery object.
+    mystery.room = randomSelector(rooms)
+
+    // This will change the background color of the card to the one connected 
+    //to the chosen killer and show the full name of the killer.
+    const theRoomName = document.getElementById("roomName")
+
+    theRoomName.innerHTML = "Room: " + mystery.room
+}
+
+console.log(mystery);
 
 // CREATE A FUNCTION revealMystery that will be invoked when you click that button. It should show something like:
 // 'The murder was committed by Jacob Green, in the living room with a rope.'
+const revealMystery = () => {
+    const theMysteryReveal = document.getElementById("mystery")
+
+
+    if (killerPicked && weaponPicked && roomPicked === true) {
```

Comment:
Here's a trick! `(killerPicked && weaponPicked && roomPicked === true)` and `(killerPicked && weaponPicked && roomPicked)` is the same. If `true` is set as a boolean (as it should be).